edx/edx-app-android
# Fix Jacoco code coverage report issue

### Description

[LEARNER-7793](https://openedx.atlassian.net/browse/LEARNER-7793)
- Fix Jacoco code coverage report issue 
- Enable your code coverage for unit tests by adding a Gradle hook.
- Exclude auto-generated code, code from the library, test classes.

Ref: https://medium.com/@android2ee/playing-with-gradle-3-and-some-few-code-coverage-on-android-with-jacoco-47df7c9328ae
**How to test**
Execute the following command in the terminal, after the completion check Jacoco report in the `[root_project]/[project_name]/build/jacoco/` path.
```java
./gradlew jacocoTestProdDebugUnitTestReport
```
